inspirehep/base/templates/format/record/Inspire_Default_HTML_brief.tpl

```diff
@@ -47,11 +47,9 @@
                 <div class="brief-record-details">
                 <div class="authors">
                   {{ render_record_authors(is_brief=true, show_affiliations=false) }}
-                  {% if record.get('earliest_date') %}
-                    <span id="record-date">
-                      - {{ record.get('earliest_date').split('-')[0] }}
-                    </span>
-                  {% endif %}
+                  <span id="record-date">
+                    - {{ record.get('earliest_date')|format_date() }}
```

Comment:
Do you need to call the filter or is it enough to just pass it like other filters `record.get('earliest_date') | format_date`.